python/cudf/cudf/core/frame.py

```diff
@@ -304,6 +304,70 @@ def _unaryop(self, op):
             result._data[name] = col.unary_operator(op)
         return result
 
+    def interleave_columns(self):
+        """
+       Interleave Series columns of a table into a single column.
+
+        Converts the column major table `cols` into a row major column.
+        Parameters
+        ----------
+        cols : input Table containing columns to interleave.
+
+        Example
+        -------
+        >>> df = DataFrame([['A1', 'A2', 'A3'], ['B1', 'B2', 'B3']])
+        >>> df
+        0    [A1, A2, A3]
+        1    [B1, B2, B3]
+        >>> df.interleave_columns()
+        0    A1
+        1    B1
+        2    A2
+        3    B2
+        4    A3
+        5    B3
+
+        Returns
+        -------
+        The interleaved columns as a single column
+        """
+
+        return self._constructor_sliced(
+            libcudfxx.reshape.interleave_columns(self)
+        )
+
+    def tile(self, count):
+        """
+        Repeats the rows from `self` DataFrame `count` times to form a
+        new DataFrame.
+
+        Different from DataFrame.tile() which repeats the DataFrame
+        sequentially with the filling function. This repeats the
+        columns by "count" number of times.
```

Comment:
You removed the implementation in `DataFrame.tile()`, is this different or the same?

Reply:
I was concerned that they were slightly different in operation but when we paired for a bit we determined this was the correct course since this tile function is libcudf++ and using frame it's deployed more coherently. The tests pass. I'll give it another close look to make sure though.

Reply:
I think this just needs a docstring update then 😄